### ISSUE TITLE
How to fix medic bots? 

### ISSUE THREAD

**Issue by mrballgaming** [nullworks/cathook#1444]
(Im only making a new issue because the last one got a little out of hand) Hello. I got a new pc for hosting cathook bots on and I'm having some trouble. Medic bots are healing each other and I have no idea how to fix it. EngineerGaming69 said i could change the health priority to 100 and it would fix it but I had no luck. I think this is caused by the medics being very close together (not that close but close enough for them to accidentally heal each other) but I'm not sure. Medic bots are just broken all together.

tl;dr: medics heal each other (this is the topic)
